src/components/LoadDatasetModal.js

```diff
@@ -55,15 +54,11 @@ class LoadDatasetModal extends Component {
     dispatchAddDataset: PropTypes.func.isRequired,
   };
 
-  buildDefaultDatasetName = (newFileLocation) => {
-    return path.basename(newFileLocation);
-  };
-
   handleLocationInput = (event) => {
     const fileLocation = event.target.value;
     this.setState({
       fileLocation,
-      defaultDatasetName: this.buildDefaultDatasetName(fileLocation),
+      defaultDatasetName: this.getNameFromLocation(fileLocation),
```

Comment:
Function `getNameFromLocation` doesn't belong to `this` and should be imported instead